lib/bootloader_setup.pm

```diff
@@ -249,7 +250,24 @@ sub boot_local_disk {
         if (match_has_tag 'inst-slof') {
             diag 'specifying local disk for boot from slof';
             my $slof = 5;
-            $slof += 1 if get_var('VIRTIO_CONSOLE');
+            if (get_var('VIRTIO_CONSOLE') && is_ppc64le) {
+                type_string_very_slow "devalias";
```

Comment:
Not sure whether here support '\n' as return, if support then don't need send key 'ret' seperately.